use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,484 @@
+$ export KATA_KERNEL_LOCATION=~/kata
+$ export KATA_ROOTFS_LOCATION=~/kata
+```
+
+## Prepare the Clear Linux Host
+
+The host could be a bare metal instance or a virtual machine. If using a 
+virtual machine, make sure that KVM nesting is enabled.  All of the 
+instructions below reference an Intel QAT.  Some of the instructions must be 
+modified if using a different Intel QAT device. You can identify the Intel QAT
+chipset by executing the following.
+
+### Identify which PCI Bus the Intel QAT card is on
+
+```sh
+$ for i in 0434 0435 37c8 1f18 1f19; do lspci -d 8086:$i; done
+```
+
+### Install necessary bundles for Clear Linux and disable kernel updates
+
+The Clear Linux installer generally installs the `os-core`, `os-core-update`, 
+and `openssh-server` bundle. Add the following minimum bundles including the 
+Linux kernel headers to complete the rest of this guide. The Linux kernel 
+headers are needed to compile the out of tree kernel modules to support Intel 
+QAT on the host. The `os-clr-on-clr` is a large development bundle that can be 
+installed instead to include everything below with additional items. This 
+guide installs the Linux LTS kernel version `4.19` from 2018 because the 
+crypto modules are set to `m` and not `y` which allows them to be replaced 
+with the out of tree driver. Otherwise you will need to build your own Clear 
+Linux kernel with the QAT drivers compiled in.
+
+```sh
+$ sudo swupd autoupdate --disable
+$ sudo swupd bundle-add network-basic kernel-lts2018 linux-lts-dev linux-dev make c-basic go-basic containers-virt dev-utils devpkg-elfutils devpkg-systemd
+$ sudo systemctl start docker
+$ sudo systemctl enable docker
+$ sudo -E gpasswd -a $USER docker
+```
+
+### Replace Clear Linux kernel with LTS kernel
+
+```sh
+$ lts_kernel=$(sudo clr-boot-manager list-kernels | grep lts2018 | head -n 1)
+$ sudo clr-boot-manager set-kernel $lts_kernel
+$ sudo reboot
+```
+
+### Allow insecure modules to load to host
+
+```sh
+$ sudo mkdir -p /etc/kernel/cmdline.d
+$ cat << EOF | sudo tee /etc/kernel/cmdline.d/load-modules.conf
+module.sig_unenforce
+EOF
+$ sudo clr-boot-manager update
+$ sudo reboot
+```
+
+### Download Intel QAT drivers
+
+This will download the Intel QAT drivers from [01.org](https://01.org/intel-quickassist-technology). 
+Make sure to check the website for the latest version
+
+```sh
+$ mkdir -p $QAT_SRC
+$ cd $QAT_SRC
+$ curl -O $QAT_DRIVER_URL
+$ QAT_DRIVER_NAME=$(echo $QAT_DRIVER_URL | rev | cut -d"/" -f1 | rev)
+$ tar xzvf $QAT_DRIVER_NAME
+```
+
+### Compile Intel QAT drivers against host kernel 
+
+The Linux kernel headers bundle `linux-dev` should already have been added. 
+This replaces any existing modules provided by Clear Linux. If the Clear Linux
+kernel is updated, these modules must be rebuilt against that kernel and 
+reinstalled. By doing it this way, you no longer have to rely on your 
+distribution including the Intel QAT firmware, won’t have to remove the 
```

Comment:
You might want to split this statement and make it clearer.

Reply:
WIll update.